.travis.yml

```diff
@@ -0,0 +1,37 @@
+language: scala
+
+sudo: required
```

Comment:
Its the [other way around](https://docs.travis-ci.com/user/ci-environment/). Without sudo its a container, which starts way quicker. As we do not need sudo here, I'll change to `sudo: false`